DevSkim-DotNet/Microsoft.DevSkim/TextContainer.cs

```diff
@@ -245,125 +197,40 @@ private bool IsBetween(string text, int index, string prefix, string suffix, str
             bool result = false;
             string preText = string.Concat(text.Substring(0, index));
             int lastPrefix = preText.LastIndexOf(prefix, StringComparison.InvariantCulture);
-            if (!string.IsNullOrEmpty(inline) && lastPrefix >= 0)
+            if (lastPrefix >= 0)
             {
                 int lastInline = preText.Substring(0, lastPrefix).LastIndexOf(inline, StringComparison.InvariantCulture);
-
                 // For example in C#, If this /* is actually commented out by a //
-                if (lastInline >= 0 && lastInline < lastPrefix && !preText.Substring(lastInline, lastPrefix - lastInline).Contains('\n'))
-                {
-                    lastPrefix = -1;
-                }
-            }
-            if (lastPrefix >= 0)
-            {
-                var commentedText = text.Substring(lastPrefix);
-                int nextSuffix = commentedText.IndexOf(suffix, StringComparison.Ordinal);
-
-                // If the index is in between the last prefix before the index and the next suffix after that
-                // prefix Then it is commented out
-                if (lastPrefix + nextSuffix > index)
-                    result = true;
-            }
-
-            return result;
-        }
-
-        /// <summary>
-        ///     Returns all boundaries matching the pattern in a text
-        /// </summary>
-        /// <param name="pattern"> Search pattern </param>
-        /// <param name="text"> Text </param>
-        /// <returns> List of boundaries </returns>
-        private List<Boundary> MatchPattern(SearchPattern pattern, string text)
-        {
-            List<Boundary> matchList = new List<Boundary>();
-
-            RegexOptions reopt = RegexOptions.None;
-            if (pattern.Modifiers != null && pattern.Modifiers.Length > 0)
-            {
-                reopt |= (pattern.Modifiers.Contains("i")) ? RegexOptions.IgnoreCase : RegexOptions.None;
```

Comment:
Did a file search for alternate location of RegexOptions.IgnoreCase support and didn't see here -maybe it's in another file now.  Just verifying we can still support.

Reply:
found it in WithinOperationDelegate elsewhere -never mind